.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
+before_install:
+- npm install -g npm@latest
```

Comment:
First I've heard of that, but I don't think I've updated npm in any of my io.js projects. will investigate